cleantalk.php

```diff
@@ -422,6 +422,14 @@ function apbct_alt_session__save__WP_AJAX()
 );
 new  \Cleantalk\Antispam\Integrations($apbct_active_integrations, (array)$apbct->settings);
 
+// Mailoptin. Pass without action because url for ajax request is domain.com/any-page/?mailoptin-ajax=subscribe_to_email_list
+if (
+    sizeof($_POST) > 0 &&
+    Get::get('mailoptin-ajax') === 'subscribe_to_email_list'
```

Comment:
Please, add an one more factor to detect the mailoptin's request - check the plugin is active.